.circleci/config.yml

```diff
@@ -2,6 +2,31 @@ version: 2.1
 
 jobs:
 
+  run-formatter:
+    docker:
+      - image: checkstyle/idea-docker:jdk11-idea2022.3.3
+
+    steps:
+      - checkout
+      - run:
+          name: Print versions
+          command: |
+            echo "Maven version:"
+            mvn --version
+            echo "Java version:"
+            java --version
+            echo "IDEA version:"
+            echo $IDEA_VERSION
```

Comment:
```suggestion
            echo "IDEA version: $IDEA_VERSION"
```
is this possible?

Reply:
@Anmol202005 , please apply this change

Reply:
hmm, this is liketo be be for consistency with  versions printed before